### ISSUE TITLE
UIWebView Release 47 Regression

### ISSUE THREAD

**Issue by edvv** [Microsoft/WinObjC#2695]
For release 0.2.170510:

`- [UIWebView loadHTMLString:baseURL:]`

throws an exception if the delegate is set (is non-nil).

That did not happen in previous releases.

**Comment by rajsesh-msft** [comment]
@edvv I am unable to repro this.  I changed woccatalog page with webview to use loadHTMLString: and it shows the string fine.  Can you provide a short repro?

**Comment by jboich** [comment]
Hi, @rajsesh-msft . I also have Access violation exception when try to use 
`[UIWebView loadHTMLString:baseURL:]` 

The issue takes place if you try to load page from a file:

```
NSString* htmlPath = [[NSBundle mainBundle] pathForResource:@"ExampleApp" ofType:@"html"];
NSString* appHtml = [NSString stringWithContentsOfFile:htmlPath encoding:NSUTF8StringEncoding error:nil];
NSURL *baseURL = [NSURL fileURLWithPath:htmlPath];
 [webView loadHTMLString:appHtml baseURL:baseURL];
```

**Comment by edvv** [comment]
@rajsesh-msft Please see the attached file/project.

[webviewbug.zip](https://github.com/Microsoft/WinObjC/files/1005261/webviewbug.zip)

**Comment by rajsesh-msft** [comment]
This is probably because this change https://github.com/Microsoft/WinObjC/commit/9c96cf25d3dad76fe60381c785a3977ea7d493ae didnt get cherry picked.

**Comment by rajsesh-msft** [comment]
Closed with #2710
